README.md

```diff
@@ -1,17 +1,15 @@
+# Ikarus
 [![Windows-MinGW-GCC](https://github.com/IkarusRepo/Ikarus/actions/workflows/windowsMinGW.yml/badge.svg)](https://github.com/IkarusRepo/Ikarus/actions/workflows/windowsMinGW.yml)
 [![Ubuntu](https://github.com/IkarusRepo/Ikarus/actions/workflows/ubuntu.yml/badge.svg)](https://github.com/IkarusRepo/Ikarus/actions/workflows/ubuntu.yml)
 [![codecov](https://codecov.io/gh/IkarusRepo/Ikarus/branch/master/graph/badge.svg?token=zJgggitPMc)](https://codecov.io/gh/IkarusRepo/Ikarus)
 [![Ubuntu-Sanitizers](https://github.com/IkarusRepo/Ikarus/actions/workflows/sanitizers.yml/badge.svg)](https://github.com/IkarusRepo/Ikarus/actions/workflows/sanitizers.yml)
 [![CodeStyle](https://github.com/IkarusRepo/Ikarus/actions/workflows/style.yml/badge.svg)](https://github.com/IkarusRepo/Ikarus/actions/workflows/style.yml)
 [![Docs](https://github.com/IkarusRepo/Ikarus/actions/workflows/ghpages.yml/badge.svg)](https://github.com/IkarusRepo/Ikarus/actions/workflows/ghpages.yml)
 [![CodeFactor](https://www.codefactor.io/repository/github/ikarusrepo/ikarus/badge/master)](https://www.codefactor.io/repository/github/ikarusrepo/ikarus/overview/master)
-# Ikarus
- 
- 
-   
+[![BCH compliance](https://bettercodehub.com/edge/badge/IkarusRepo/Ikarus?branch=master)](https://bettercodehub.com/)
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/5c588e67d1e541fc9be3c7377297aa8a)](https://www.codacy.com/gh/IkarusRepo/Ikarus/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=IkarusRepo/Ikarus&amp;utm_campaign=Badge_Grade)
 
+This project tries to provide a easy to read and easy to use finite element framework.
+It is heavily inspired by [DUNE](https://dune-project.org/), the book [DUNE — The Distributed and Unified Numerics Environment](https://www.springer.com/gp/book/9783030597016), [deal.II](https://www.dealii.org/), [Kratos](https://github.com/KratosMultiphysics/Kratos) and [Autodiff](https://autodiff.github.io/).
 
-  
-  
-  
-  
+Our documentation resides at https://ikarusrepo.github.io/.
```

Comment:
Codacy has a fix for the issue: [Bare URL used](https://app.codacy.com/gh/IkarusRepo/Ikarus/pullRequest?prid=7699962)
```suggestion
Our documentation resides at <https://ikarusrepo.github.io/>.
```